db/dataGenerator.js

```diff
@@ -0,0 +1,48 @@
+const Sequelize = require('sequelize');
+
+const sequelize = new Sequelize('booking', 'yerin', 'yerin', {
```

Comment:
git ignore configs for your DB

Reply:
Okay, I will change my password of database and add a config file for db. Thanks!